### DISCUSSION TITLE
Does Veramo provides "peer wise DID support"

### DISCUSSION THREAD

**Discussion by devrajsinghrawat** [decentralized-identity/veramo#402] [Q&A]
Hi Team,

DIF standard provides the specification for Peer wise DID specification as mentioned in the below link.
https://identity.foundation/peer-did-method-spec/

Does Veramo provides support for the same.

If not, then is there any future plans for the same.

if yes, then how can we use these features.

Regards
Dev

**Comment by mirceanis** [comment]
Hi, Dev
This feature does not exist in veramo right now.
We are considering a "connections" API that would deal with maintaining message or reply channels to different entities.
Peer-wise DIDs would fall under that section.
However, this is only in the discussions phase and not very high on the priority list.

**Reply by devrajsinghrawat** [reply]
Hi @mirceanis , Thanks for your response

When we say "a "connections" API that would deal with maintaining message or reply channels to different entities."

Does that mean , for every interaction a new DID will be created? 

Regards
Dev

**Reply by mirceanis** [reply]
It's too early to tell, and it's not clear to us either if we will support peer DIDs directly or use some other model.

**Reply by mirceanis** [reply]
If someone wanted to create a plugin for veramo to create, manage, and resolve peed DIDs, that is probably possible already.

**Reply by devrajsinghrawat** [reply]
many thanks for the details
